pilot/pkg/serviceregistry/external/conversion.go

```diff
@@ -36,6 +39,104 @@ func convertPort(port *networking.Port) *model.Port {
 	}
 }
 
+// ServiceToServiceEntry converts from internal Service representation to ServiceEntry
+// This does not include endpoints - they'll be represented as EndpointSlice or EDS.
+//
+// See convertServices() for the reverse conversion, used by Istio to handle ServiceEntry configs.
```

Comment:
If we have a reverse, maybe we should test a round trip is lossless (if its supposed to be)

Reply:
It's not - and I don't think it's worth the cost. 
ServiceEntry allows a list of hosts. The internal registry has one entry per host.